MIGRATION-4.0.md

```diff
@@ -0,0 +1,81 @@
+# Migrating from v3 to v4
+
+## In theory...
+
+API has been deeply reviewed to both simplify configuration and improve overall performances. We didn't take care of backward compatibility during this major changes. Hence, you will probably need to rewrite your current scripts to make it work with this fourth version.
```

Comment:
s/API has been deeply reviewed to both simplify configuration and improve overall performances. We didn't take care of backward compatibility during this major changes. Hence, you will probably need to rewrite your current scripts to make it work with this fourth version./The API has been deeply reviewed to both simplify configuration, and improve overall performances. We chose to break backward compatibility to allow the implementation of the new features. That means you will probably need to rewrite your current scripts to make them work with the new version./